x-pack/packages/security-solution/upselling/pages/attack_discovery/index.tsx

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { KibanaPageTemplate } from '@kbn/shared-ux-page-kibana-template';
+import { EuiPageHeader, EuiSpacer } from '@elastic/eui';
+import React, { useMemo } from 'react';
+
+import { PageTitle } from './page_title';
+import { AttackDiscoveryUpsellingSection } from '../../sections/attack_discovery';
+
+interface Props {
+  actions?: React.ReactNode;
+  availabilityMessage: string;
+  upgradeMessage: string;
+}
+
+/**
+ * This component handles the styling of the _page_ that hosts the `AttackDiscoveryUpsellingSection`
+ */
+const AttackDiscoveryUpsellingPageComponent: React.FC<Props> = ({
+  actions,
+  availabilityMessage,
+  upgradeMessage,
+}) => {
+  const pageTitle = useMemo(() => <PageTitle />, []);
```

Comment:
Is this `useMemo` doing any optimization? `PageTitle` does not receive any props, and has already been memoized.

Reply:
This is awesome, thanks for doing this experiment, I always lear something when reviewing your PRs 😊 
 
Thanks! resolving topic.